karaf/apache-brooklyn/src/main/resources/etc/org.apache.brooklyn.classrename.cfg

```diff
@@ -0,0 +1,21 @@
+################################################################################
```

Comment:
This file should probably be added via the brooklyn `feature.xml` also to make extension / upgrade by downstream projects easier
```
<feature name="example-config" version="${project.version}">
        <configfile finalname="${karaf.etc}/${groupId}.classrename.cfg" override="true">
            mvn:${project.groupId}/${project.artifactId}/${project.version}/cfg/classrename
        </configfile>
</feature>
```